### DISCUSSION TITLE
Overflow and hover tooltip

### DISCUSSION THREAD

**Discussion by ShaungBhone** [tailwindlabs/tailwindcss#14626] [Help]
How can I prevent the tooltip from going behind the table? I need that overflow to handle that table and ensure responsiveness.

https://github.com/user-attachments/assets/08408343-fad8-494b-82fa-8ec7d9807303

**Comment by wongjn** [comment]
You could consider wrapping the tooltip in a [`<Teleport>` component](https://vuejs.org/guide/built-ins/teleport) so that the tooltip is not a child of the `overflow-x-auto` element.

**Reply by ShaungBhone** [reply]
I'm using vue2.

**Reply by wongjn** [reply]
You could consider using a Vue 2 equivalent.
